config/settings.yml

```diff
@@ -2,7 +2,7 @@ dor_services:
   url: 'https://fakeuser:fakepass@fakeserver.example.edu/dor'
 workflow:
   url: 'https://workflows.example.org'
-  # logfile: 'log/wfs/workflow_service.log'
+  logfile: 'log/wfs/workflow_service.log'
```

Comment:
@ndushay This commit needs to go away or include other changes. This is intentionally left unset because the way we initialize WFS is to pass it an existing logger instance.  I.E., that will take precedence over this config.

Reply:
ah - so should we remove this possibility altogether?  (no commented out line)

Reply:
Or change the way we initialize WFS in conjunction w/ this.  Definitely avoiding having config that is "active" but useless helps prevent confusion.